docs/arlas-api-exploration.md

```diff
@@ -261,6 +262,49 @@ In the case of `lt`, `gt`, `lte`, `gte`, `range` operations that are applied on
 
 > Example: `curl --header "Partition-Filter: {f":[[{"field":"city","op":"eq","value":"Bordeaux"}]]}" https://api.gisaia.com/demo/arlas/explore/cities/_count`
 
+#### Column filtering
+
+A coma-separated list of columns can be passed in request header `column-filter`. Wildcards are supported.
+
+A column filter stands for the fields that are available to a request body:
+- if a request body field doesn't belong to the column filter, a 403 is returned with the message `The field '%s' isn't available` or `The fields '%s' aren't available`;
+- only fields that belong to the column filter can be returned. 
+
+Examples of `column-filter`:
+
+- `params.city,params.country` make available `params.city` and `params.country` 
+- `params`, `params*`, `params.*`, `*params` make available `params.city`, `params.country`, `params.weight`, and so on.
+- `*` make all fields available
+- `*.*` make only subfields available, e.g. `params.city` and `params.country` but not `id`
+
+If no column filter, or a blank column filter is provided, then it isn't eventually used.
+
+The following endpoints use this header:
+
+| Endpoint | Filtering result |
+| --------------------------------------------------- | --------------------------------------------------- |
+| /arlas/explore/**_list**                 | Only fields matching this filter will be returned |
+| /arlas/explore/`{collection}`/**_describe** | Only fields matching this filter will be returned |
+| /arlas/explore/`{collection}`/**_count** | Return a 403 if one of the filter fields is not in the column filter |
+| /arlas/explore/`{collection}`/**_range** | Return a 403 if the field, or one of the filter fields, is not in the column filter. Otherwise only fields matching the filter will be returned |
+| /arlas/explore/`{collection}`/**_search** | Return a 403 if one of the filter, projection and page fields is not in the column filter. Otherwise only fields matching the filter will be returned |
+| /arlas/explore/`{collection}`/**_geosearch** | Return a 403 if one of the filter, projection or page fields is not in the column filter. Otherwise only fields matching the filter will be returned |
+| /arlas/explore/`{collection}`/**_geosearch/`{z}`/`{x}`/`{y}`** | Return a 403 if one of the filter, projection or page fields is not in the column filter. Otherwise only fields matching the filter will be returned |
+| /arlas/explore/`{collections}`/**_aggregate** | Return a 403 if one of the aggregation or filter fields is not in the column filter |
+| /arlas/explore/`{collections}`/**_tile**/`{z}`/`{x}`/`{y}`.png | Return a 403 if one of the aggregation or filter fields is not in the column filter |
```

Comment:
`one of the filter, projection or page fields is not in the column filter` instead of `one of the aggregation or filter fields`